docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+    )
+
+def add_customer_button5() -> rx.Component:
+    return rx.dialog.root(
+        rx.dialog.trigger(
+            rx.button(
+                rx.icon("plus", size=26),
+                rx.text("Add User", size="4"),
+            ),
+        ),
+        rx.dialog.content(
+            rx.dialog.title(
+                "Add New User",
+            ),
+            rx.dialog.description(
+                "Fill the form with the user's info",
+            ),
+            rx.form(
+                rx.flex(
+                    rx.input(
+                        placeholder="User Name", name="name", required=True
+                    ),
+                    rx.input(
+                        placeholder="user@reflex.dev",
+                        name="email",
+                    ),
+                    rx.select(
+                        ["Male", "Female"],
+                        placeholder="male",
+                        name="gender",
+                    ),
+                    rx.flex(
+                        rx.dialog.close(
+                            rx.button(
+                                "Cancel",
+                                variant="soft",
+                                color_scheme="gray",
+                            ),
+                        ),
+                        rx.dialog.close(
+                            rx.button(
+                                "Submit", type="submit"
+                            ),
+                        ),
+                        spacing="3",
+                        justify="end",
+                    ),
+                    direction="column",
+                    spacing="4",
+                ),
+                on_submit=State5.add_user,
+                reset_on_submit=False,
+            ),
+            max_width="450px",
+        ),
+    )
+
+def graph5():
+    return rx.recharts.bar_chart(
+        rx.recharts.bar(
+            data_key="value",
+            stroke=rx.color("accent", 9),
+            fill=rx.color("accent", 8),
+        ),
+        rx.recharts.x_axis(data_key="name"),
+        rx.recharts.y_axis(),
+        data=State5.users_for_graph,
+        width="100%",
+        height=250,
+    )
+```
+
+```python eval
+rx.vstack(
+        add_customer_button5(),
+        rx.table.root(
+            rx.table.header(
+                rx.table.row(
+                    rx.table.column_header_cell("Name"),
+                    rx.table.column_header_cell("Email"),
+                    rx.table.column_header_cell("Gender"),
+                ),
+            ),
+            rx.table.body(
+                rx.foreach(
+                    State5.users, show_user5
+                ),
+            ),
+            variant="surface",
+            size="3",
+            width="100%",
+        ),
+        graph5(),
+        align="center",
+        width="100%",
+        on_mouse_enter=State5.transform_data
+    )
+```
+
+```python
+import reflex as rx
```

Comment:
Maybe would be less cluttered to put the code inside a drop down? Or I guess maybe your big point is that the code is so short that you wouldn't need to do that.

Reply:
waiting for another PR in reflex web for this to be possible, let's maybe do it as a follow up